src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight: function(c = 1) {
+      if (c > 0) {
+        this.coords.x += c;
+      }
+
+      return this;
+    },
```

Comment:
don't use the function for method, use simplified syntax
```suggestion
    goRight(c = 1) {
      if (c > 0) {
        this.coords.x += c;
      }

      return this;
    },
```